specs/subresourceintegrity/index.html

```diff
@@ -427,6 +430,18 @@ <h4 id="is-varresourcevar-eligible-for-integrity-validation">Is <var>resource</v
 resources accessed over a <code>file</code> scheme URL are unlikely to be
 eligible for integrity checks.</p>
 
+      <p>One should note that being a <a href="#dfn-privileged-context">privileged context</a> (e.g. HTTPS) is not
+necessary for the use of integrity validation. Because resource integrity is
+only an application level security tool, and it does not change the security
+state of the user agent, a privileged context is unnecessary. However, if
+integrity is used in an <a href="#dfn-unprivileged-context">unprivileged context</a> (e.g. HTTP), users should be
+aware that the integrity provides <em>no security guarantees</em> at all. Even
+if the integrity is used in a privileged context, but on a resource that is not
+potentially secure, then privacy is at great risk. For these reasons, authors
```

Comment:
Is "potentially secure" defined anywhere?

I assume you use "potentially secure" because HTTPS could be insecure if the cert verification fails?